system/core/HyphaRequest.php

```diff
@@ -149,11 +145,7 @@ public function getScheme() {
 
 		public function getHost() {
 			return $_SERVER['HTTP_HOST'];
-		}
-
-		public function getPort() {
-			return $_SERVER['SERVER_PORT'];
-		}
+		}	
```

Comment:
This line adds trailing whitespace, it seems (and would not otherwise need to be changed).